net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -303,11 +303,15 @@ EOF;
 	// Create rc.d file
 	$rc_file_stop = <<<EOF
 if [ -e /var/run/quagga/zebra.pid ]; then
-	/bin/kill -9 `/bin/cat /var/run/quagga/zebra.pid`
+	# XXX: See Bug #6305 - Quagga 1.x does not like SIGKILL
+	#/bin/kill -9 `/bin/cat /var/run/quagga/zebra.pid`
+	/bin/kill `/bin/cat /var/run/quagga/zebra.pid`
```

Comment:
Please just remove kill -9 line instead of commenting it out. While here, just replace /bin/kill by pkill -F